Voice Calculator/voiceCalc.py

```diff
@@ -0,0 +1,107 @@
+import os
+import pyttsx3
+import speech_recognition as sr
+import tkinter.messagebox as tmessage
+import wolframalpha
+
+
+from os.path import exists
+
+
+listener = sr.Recognizer()
+engine = pyttsx3.init()
+voices = engine.getProperty('voices')
+engine.setProperty('voice', voices[0].id)
+wolfprimeaplahe_app = '9RQ7T2-WKUWLR74WJ'
```

Comment:
why you have exposed the api key?

Reply:
Because if I would not then the code would not have ran...